File: Witcher-front/src/Components/Witcher.js
import React from 'react';

const divStyle = {
    width: '800px',
    border: '2px #000',
    borderRadius: '10px'
};

class Witcher extends React.Component {

    constructor(props) {
        super(props);
    }

    render() {
        return(
            <div style={divStyle}>
                <table border="1px">
                {this.props.array.map((elem) => {
                    return (
                        <tr>
                            <td>Имя : {elem.witcherName}</td>
                            <td>Здоровье : {elem.health}</td>
                            <td>Деньги : {elem.money}</td>
                            <td>Меч : {elem.swordName}</td>
                            <td>Броня : {elem.armorName}</td>
                            <td>Харизма : {elem.charisma}</td>
                            <td>Локация : {elem.locationName}</td>
                        </tr>
                    );})}
                </table>
            </div>
        );
    }

}

export default Witcher;

File: Witcher-back/src/main/java/web/witcher/Entities/Witcher.java
package web.witcher.Entities;



import lombok.Data;

import javax.persistence.*;

@Entity
@Data
@Table(name = "Witcher")
public class Witcher {

    @Id
    @Column(name = "witcherName")
    private String witcherName;

    @Column(name = "armorName")
    private String armorName;
    
    @Column(name = "swordName")
    private String swordName;

    @Column(name = "potionName")
    private String potionName;
    
    @Column(name = "oilName")
    private String oilName;
    
    @Column(name = "health")
    private int health;
    
    @Column(name = "money")
    private int money;
    
    @Column(name = "charisma")
    private int charisma;
    
    @Column(name = "locationName")
    private String locationName;
}
